bam_masterdata/cli/cli.py

```diff
@@ -38,20 +38,44 @@ def cli():
     it is using the value of the `OPENBIS_URL` environment variable.
     """,
 )
-def fill_masterdata(url):
+@click.option(
+    "--excel-file",
+    type=click.Path(exists=True, dir_okay=False),
+    required=False,
+    help="""
+    (Optional) The path to the Masterdata Excel file.
+    """,
+)
+def fill_masterdata(url, path):
```

Comment:
you have to change path to excel_file in this function

maybe click has another option for aliases if you prefer so